### PR TITLE
feature/account-problem - Data is updated when saving

### PR DESCRIPTION
**Pull request by AgusSosa2001**

### CODE REVIEW

File: admin/templates/components/navbar.php [hunk 1]
```diff
@@ -74,6 +74,12 @@
       </div>
     </div>
   </nav>
+<<<<<<< HEAD
+<<<<<<< Updated upstream
+</div>    
+=======
+=======
+>>>>>>> 2921f79bb7f537ab02f8b2f09a4d3595a30bfb37
```
Comment (Miguelslo27) [inline admin/templates/components/navbar.php]: There is a conflict here

File: admin/templates/components/navbar.php [hunk 2]
```diff
@@ -110,4 +116,9 @@
     <?php setGlobal('actualItem', $conf) ?>
     <?php getTemplate('components/navbar-item') ?>
   </div>
+<<<<<<< HEAD
+</div>
+>>>>>>> Stashed changes
+=======
 </div>    
+>>>>>>> 2921f79bb7f537ab02f8b2f09a4d3595a30bfb37
```
Comment (Miguelslo27) [inline admin/templates/components/navbar.php]: Another conflict

File: core/config_dev.php [hunk 3]
```diff
@@ -80,5 +80,6 @@
 
 define('ADMIN_TEMPLATE', 'pages/dashboard');
 define('ADMIN_TEMPLATE_CONFIG', 'pages/configuration');
+define('ADMIN_TEMPLATE_CONFIG', 'pages/configuration');
```
Comment (Miguelslo27) [inline core/config_dev.php]: You have repeated the `ADMIN_TEMPLATE_CONFIG` with the exact same value

**Review by Miguelslo27 — COMMENTED** [review]
Just need to fix conflicts, the rest seems to be ok 👌

File: admin/templates/components/navbar.php [hunk 4]
```diff
@@ -74,40 +74,43 @@
...
+  <?php setGlobal('actualItem', $users) ?>
+  <?php getTemplate('components/navbar-item') ?>
+  <!-- ORDERS BOX -->
+  <?php setGlobal('actualItem', $orders) ?>
+  <?php getTemplate('components/navbar-item') ?>
+  <!-- CONFIGURATION BOX -->
+  <?php setGlobal('actualItem', $conf) ?>
+  <?php getTemplate('components/navbar-item') ?>
+</div>
+</div>
```
Comment (fdsosa) [inline admin/templates/components/navbar.php]: Esta etiqueta `</div>` no esta cerrando nada. Hay que elimanarla

File: admin/templates/components/navbar.php [hunk 5]
```diff
@@ -74,40 +74,43 @@
...
+
+</div>
```
Comment (fdsosa) [inline admin/templates/components/navbar.php:113]: Lo mismo que la anterior

File: admin/templates/components/navbar.php (hunk 4 above)
Comment (Miguelslo27) [reply admin/templates/components/navbar.php]: @AgusSosa2001 This is unresolved yet

File: admin/templates/components/navbar.php (hunk 5 above)
Comment (Miguelslo27) [reply admin/templates/components/navbar.php:113]: @AgusSosa2001 same here

**Pull request merged**